ui/src/components/charts/PrimarySiteChart.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable */
 import React from 'react';
 import { css } from '@emotion/react';
-// import { addInSQON } from '@arranger/components/dist/SQONView/utils';
+import { addInSQON } from '@overture-stack/arranger-components/dist/SQONViewer/utils';
```

Comment:
down the line we must replace this import with one from SQONbuilder(https://github.com/overture-stack/sqon-builder/). I know this code existed already, but it's not good practice importing things digging into the packages' folders

Reply:
Yes, I had a feeling there were better places to obtain some of the SQON code, thank you

Reply:
On review though SQON Builder isn't a 1:1 same exports as used here so maybe we create a follow up ticket or add to an existing ticket

Reply:
oh yeah. further down the line. definitely not something for this PR